searx/enginelib/__init__.py

```diff
@@ -76,7 +76,7 @@ class Engine:  # pylint: disable=too-few-public-methods
 
     # settings.yml
 
-    categories: List[str]
+    categories: list[str]
```

Comment:
This changed form 'List` to `list` - I believe this breaks Python 3.8? I just had this issue in my PR.

Reply:
Hi @glanham-jr, you can add `from __future__ import annotations` to get [Shorthand Syntax](https://typing.readthedocs.io/en/latest/source/best_practices.html#shorthand-syntax) in py3.8 / see my comment above: https://github.com/searxng/searxng/pull/3330#issuecomment-1993824738